bundles/org.openhab.binding.airq/README.md

```diff
@@ -0,0 +1,225 @@
+| Parameter              | Description                   |
+|------------------------|-------------------------------|
+| id                     | Device ID                     |
+| hardwareVersion        | Hardware version              |
+| softwareVersion        | Firmware version              |
+| sensorList             | Available sensors             |
+| sensorInfo             | Information about the sensors |
+| industryVersion        | Industry version              |
+
+## Channels
+
+The air-Q Thing offers access to all sensor data of the air-Q, according to its version.
+This includes also the Maximum Error per sensor value.
+For the Maximum Error channels just add `_maxerr` to the channel names.
+
+| channel                   | type     | description                                                         |
+|---------------------------|----------|---------------------------------------------------------------------|
+| Status                    | String   | Status of the sensors                                               |
+| avgFineDustSize           | Number   | Average size of Fine Dust [experimental]                            |
+| bat                       | Number   | Battery State                                                       |
+| fineDustCnt00_3           | Number   | Fine Dust >0,3 &mu;m                                                |
+| fineDustCnt00_5           | Number   | Fine Dust >0,5 &mu;m                                                |
+| fineDustCnt01             | Number   | Fine Dust >1 &mu;m                                                  |
+| fineDustCnt02_5           | Number   | Fine Dust >2,5 &mu;m                                                |
+| fineDustCnt05             | Number   | Fine Dust >5 &mu;m                                                  |
+| fineDustCnt10             | Number   | Fine Dust >10 &mu;m                                                 |
+| co                        | Number   | CO concentration                                                    |
+| co2                       | Number   | CO<sub>2</sub> concentration                                        |
+| dCO2dt                    | Number   | Change of CO<sub>2</sub> concentration                              |
+| dHdt                      | Number   | Change of Humidity                                                  |
+| dewpt                     | Number   | Dew Point                                                           |
+| door_event                | Switch   | Door Event (experimental)                                           |
+| health                    | Number   | Health Index                                                        |
+| humidity                  | Number   | Humidity in percent                                                 |
+| humidity_abs              | Number   | Absolute Humidity                                                   |
```

Comment:
humidityAbsolute or asbsoluteHumidity. This has not been changed in the XML either.